cinema/serializers.py

```diff
@@ -1,13 +1,13 @@
 from rest_framework import serializers
 
-from cinema.models import Movie
+from cinema.models import Movie, Actor, Genre, CinemaHall
 
 
 class MovieSerializer(serializers.Serializer):
     id = serializers.IntegerField(read_only=True)
-    title = serializers.CharField(max_length=255, required=True)
-    description = serializers.CharField(required=True)
-    duration = serializers.IntegerField(required=True)
+    title = serializers.CharField(max_length=255)
```

Comment:
thanks for fixing the source code)

Reply:
haha, sorry :) pretty sure i was doing it before submitting , probably havnt even realised that i had been originally that way, that it isn't something that i messed up as usual. will change it back

Reply:
no need, source code is wrong)